RIOT-OS/RIOT
# tests/feature_resolution: add build system unit tests

### Contribution description

Detecting bugs in feature resolution previously resulted in a lot of manual testing and more than once an issue slipped through. Dedicated unit tests should fix both.

### Testing procedure

~~Hopefully, it is indeed as trivial as I had hoped for and Murdock will just run the test.~~ The issue described in https://github.com/RIOT-OS/RIOT/pull/15967#pullrequestreview-586857807 should be detected by this test.

Update: The test is now part of `make static-test`.

### Issues/PRs references

https://github.com/RIOT-OS/RIOT/pull/15957, https://github.com/RIOT-OS/RIOT/pull/15855
